lib/util/buttons/labeled_checkbox.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+
+class LabeledCheckbox extends StatelessWidget {
+  const LabeledCheckbox({
+    super.key,
+    required this.label,
+    required this.value,
+    required this.onChanged,
+  });
+
+  final String label;
+  final bool value;
+  final ValueChanged<bool> onChanged;
+
+  @override
+  Widget build(BuildContext context) {
+    return InkWell(
+      onTap: () {
+        onChanged(!value);
+      },
+      child: Row(
+        mainAxisSize: MainAxisSize.min,
+        children: <Widget>[
+          Checkbox(
+            value: value,
+            onChanged: (bool? newValue) {
```

Comment:
when is the bool null? and shouldn't we return in that case?

Reply:
Not my choice, the Flutter checkbox expects type `void Function(bool?)?` here. This is only relevant for tristate checkboxes, though.

Reply:
Yes but the next line would throw an error in that case because of the strict null check. And I think we should maybe handle that more gracefully.